### PR TITLE
ApiDump: Added merge code to .luacheckrc

### PR DESCRIPTION
**Pull request by Seadragon91**
The merge code will be added into the file `.luacheckrc`, that is created by APIDump.

### CODE REVIEW

File: Server/Plugins/APIDump/main_APIDump.lua [hunk 1]
```diff
@@ -1696,7 +1696,53 @@ globals =
...
+function WriteTable(a_File, a_TableName)
+	a_File:write(a_TableName, " = \n{\n")
+	for _, Entry in ipairs(_G[a_TableName]) do
+		a_File:write("\t\"", Entry, "\",\n")
+	end
+	a_File:write("}\n\n")
+end
+
+-- Load plugins's luacheck
+local FilePluginLuacheck = assert(loadfile(".plugin_luacheck"))
```
Comment (madmaxoft) [inline Server/Plugins/APIDump/main_APIDump.lua]: Oh COME ON! Really, assert? What if a plugin doesn't need any config?

Comment (Seadragon91) [inline Server/Plugins/APIDump/main_APIDump.lua]: Ops, sorry. Will correct that,

Comment (sphinxc0re) [inline Server/Plugins/APIDump/main_APIDump.lua]: And why aren't you loading the normal `.luacheck` file?

Comment (Seadragon91) [inline Server/Plugins/APIDump/main_APIDump.lua]: I load a plugin specific one, the merge code is for merging both options / tables into one file.

Not sure what you really mean.

**Pull request merged**